unittests/__init__.py

```diff
@@ -0,0 +1,23 @@
+import os
+
+import mesonbuild.compilers
+from mesonbuild.mesonlib import setup_vsenv
+
+def unset_envs():
+    # For unit tests we must fully control all command lines
+    # so that there are no unexpected changes coming from the
+    # environment, for example when doing a package build.
+    varnames = ['CPPFLAGS', 'LDFLAGS'] + list(mesonbuild.compilers.compilers.CFLAGS_MAPPING.values())
+    for v in varnames:
+        if v in os.environ:
+            del os.environ[v]
+
+def set_envs():
+    os.environ.setdefault('MESON_UNIT_TEST_BACKEND', 'ninja')
+
+def init():
+    setup_vsenv()
+    unset_envs()
+    set_envs()
+
+init()
```

Comment:
I don't really see a point in having an init() function here? Any particular reason for doing it this way?

My guess would be that it's imitating the use of main() in run_unittests.py but there's a comparatively specific reason why we do that there, and it isn't applicable to `__init__.py`.

Reply:
It's just my instinct of not calling non-pure functions in the global scope, and limiting it to one place. No other reason